src/modules/jcms_rest/src/Plugin/rest/resource/EventItemRestResource.php

```diff
@@ -67,8 +67,14 @@ public function get($id) {
         $response['uri'] = $node->get('field_event_uri')->first()->getValue()['uri'];
       }
       // Content is optional, only display if there is no Event URI.
-      elseif ($content = $this->processFieldContent($node->get('field_content'))) {
-        $response['content'] = $content;
+      else{
```

Comment:
this logic seems duplicated in all the resources, can it be extracted into a base class or a trait or similar?

Reply:
We could make it a trait but it is only 6 lines.

Reply:
Multiplied by the times it is repeated, it's 36 lines :)
Most importantly, should be easy to reuse in new kinds of entities and packaging it up in a trait makes that easier